src/app/cli/src/graphql.ml

```diff
@@ -498,12 +505,31 @@ module Types = struct
       obj "DeleteWalletPayload" ~fields:(fun _ ->
           [pubkey_field ~resolve:(fun _ key -> Stringable.public_key key)] )
 
-    let reset_trust_status =
-      obj "ResetTrustStatusPayload" ~fields:(fun _ ->
-          [ field "ipAddress" ~typ:(non_null string)
-              ~doc:"An IPv4 or IPv6 address"
+    let trust_status =
+      obj "TrustStatusPayload" ~fields:(fun _ ->
+          let open Trust_system.Peer_status in
+          [ field "trust" ~typ:(non_null float) ~doc:"Trust score"
+              ~args:Arg.[]
+              ~resolve:(fun _ {trust; _} -> trust)
+          ; field "banned_status" ~typ:string ~doc:"Banned status"
               ~args:Arg.[]
-              ~resolve:(fun _ ip_addr -> Stringable.ip_address ip_addr) ] )
+              ~resolve:(fun _ {banned; _} -> Stringable.banned_status banned)
+          ] )
+
+    let ip_trust_status =
+      obj "TrustStatusPayload" ~fields:(fun _ ->
```

Comment:
can you combine these two objects and make `ip_addr` nullable to model the above data?

Reply:
Even simpler, make `ip_addr` non-nullable, always return the associated IP address with the status. The client can choose to display that IP, or not.